### PR TITLE
feat(slack): Add check function for bot not in channel

### PR DESCRIPTION
**Pull request by Ayush0Chaudhary**

### CODE REVIEW

File: bot/slack/runner.py [hunk 1]
```diff
@@ -120,7 +125,21 @@ def run_subscribe_command(
...
+                "blocks": [{
+                    "type": "section",
+                    "text": {
+                        "type":
+                        "mrkdwn",
+                        "text":
+                        "I can not subscribed this repo to the channel,\n beacuse I am not in this channel, \nIf you need me to send you updates, invite me in this channel first."
+                    }
+                }]
+            }
```
Comment (BURG3R5) [inline bot/slack/runner.py:142]: Prettify this by using [this](https://github.com/BURG3R5/github-slack-bot/blob/5dd965c200ba2c982a98048e41a83b77198265fe/main.py#L91-L105) format and sectioning

File: bot/slack/runner.py [hunk 2]
```diff
@@ -29,11 +32,13 @@ def __init__(
         logger: Logger,
         base_url: str,
         secret: str,
+        token: str,
     ):
         SlackBotBase.__init__(self)
         self.logger = logger
         self.base_url = base_url
         self.secret = secret.encode("utf-8")
+        self.client = WebClient(token=token)
```
Comment (BURG3R5) [inline bot/slack/runner.py:41]: Remove the `client` from here and `Messenger` and instead initialize it in `SlackBotBase` so it can be used in both places

**Review by BURG3R5 — CHANGES_REQUESTED** [review]
Check comments for details

**Comment by Ayush0Chaudhary** [comment]
will do it by tomorrow 3am.

**Review by BURG3R5 — APPROVED** [review]

**Review by BURG3R5 — APPROVED** [review]

**Pull request merged**